### PR TITLE
github_pr_lifecycle_status trigger

### PR DESCRIPTION
**Pull request by shem8**
Add and example for github_pr_lifecycle_status trigger

### CODE REVIEW

File: docs/building-your-quest/triggers-and-payloads.md [hunk 1]
```diff
@@ -50,6 +50,29 @@ Generated payload:
 - `githubRepository:` The name of the repository in which the PR was opened
 - `githubWorkflowRunUrl`: Defined when `eventType` is one of `github_pr_workflow_complete_success` and `github_pr_workflow_complete_failure`. Holds a URL of the workflow run. 
 
+Here's an example of how to run the [Github Actions] and approve it once it all run successfully. Pay attention that we move the user only when they merge the PR (that's possible because the user can merge the PR only when you approve the PR)
```
Comment (esapir) [inline docs/building-your-quest/triggers-and-payloads.md:53]: What do you mean by "we move the user"?

**Review by darmalovan — APPROVED** [review]

File: docs/building-your-quest/triggers-and-payloads.md (hunk 1 above)
Comment (shem8) [reply docs/building-your-quest/triggers-and-payloads.md:53]: Calling `finish_step`

Comment (esapir) [reply docs/building-your-quest/triggers-and-payloads.md:53]: So: 1. Maybe instead of 'we', refer to the YAML. 2. Instead of `move`, maybe use `advance`?

Comment (shem8) [reply docs/building-your-quest/triggers-and-payloads.md]: good idea, fixed.

**Review by esapir — APPROVED** [review]

**Pull request merged**